apps/odyssey-stats/src/components/root-child.tsx

```diff
@@ -0,0 +1,73 @@
+import config from '@automattic/calypso-config';
+import { useLayoutEffect, useMemo, useState } from 'react';
+import ReactDOM from 'react-dom';
+import { useSelector } from 'calypso/state';
+import { isJetpackSite } from 'calypso/state/sites/selectors';
+import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import type { FunctionComponent, ReactNode } from 'react';
+
+const RootChild: FunctionComponent< { children: ReactNode } > = ( { children } ) => {
```

Comment:
I just noticed this component replacement is for the wrapper of the [Popover component](https://github.com/Automattic/wp-calypso/blob/322700bb214dfcffa06cf33d88e333c576493965/packages/components/src/popover/index.jsx#L413). Could we describe it in annotations somewhere?

Reply:
Addressed in 5623b6ade5b